lesson2/homework2_savchenko.rb

```diff
@@ -0,0 +1,30 @@
+module M
+  def self.included(base)
+    base.extend ForMetaClass
+  end
+
+  def initialize
+    puts Foo.methods - Object.methods # => [:MetaClassMethod]
```

Comment:
`#initialize`? In module? Don't do that. You need to output included class and instance methods just in `#included` hook.